parser.go

```diff
@@ -90,25 +94,52 @@ type TemplateFieldSpecifier struct {
 	Length       uint16
 }
 
+// An option can be passed to New()
+type Option func(*Session)
+
+// WithTemplateIDAliasing enables or disables template id aliasing. The default is disabled.
+func WithTemplateIDAliasing(v bool) Option {
+	return func(s *Session) {
+		s.withTemplateIDAliasing = v
+	}
+}
+
 // The Session is the context for IPFIX messages.
 type Session struct {
 	buffers *sync.Pool
 
-	mut       sync.RWMutex
-	templates [][]TemplateFieldSpecifier
-	minRecord []uint16
+	mut sync.RWMutex
+
+	withTemplateIDAliasing bool
```

Comment:
Style: too long. Perfect name for the public, exported option but internally this could equally well be `useAliasing` or something.